### DISCUSSION TITLE
Custome Page With Table

### DISCUSSION THREAD

**Discussion by Dr-Maspix** [filamentphp/filament#11867] [Help] [answered]
### Package

Other

### Package Version

v3.0.0

### How can we help you?

Dear, 

Kindly i need help with my issue , 
i created custom page to view auth user products , 

![image](https://github.com/filamentphp/filament/assets/44633806/cee4b30e-aefa-4c62-894a-223be9aeeacd)

![image](https://github.com/filamentphp/filament/assets/44633806/962fad9b-1753-4a44-9c73-385a01f2d565)

now the issue how i can make new custom page and link it with table from this custom page 
to preview single product Info ,
how i can create view action and link it with the new custom page.

Regards

**Comment by leandrocfe** [comment, marked as answer]
You could use an infolist to preview the product info:

```php
php artisan filament:install --infolists
```
https://filamentphp.com/docs/3.x/infolists/installation#existing-laravel-projects

After:

- [add an infolist to a livewire component](https://filamentphp.com/docs/3.x/infolists/adding-an-infolist-to-a-livewire-component)
```php
...
public Product $product;

public function productInfolist(Infolist $infolist): Infolist
{
    return $infolist
        ->record($this->product)
        ->schema([
            TextEntry::make('name'),
            ...
        ]);
}
```

- create a view action in the table linking it with the new custom page
```php
->actions([
    \Filament\Tables\Actions\ViewAction::make()
        ->url(fn (Product $record) => route('products.show', $record))
])
```
